src/EventSubscriber/MaintenanceEventsubscriber.php

```diff
@@ -33,6 +33,10 @@ public function handle(RequestEvent $event): void
     {
         $configuration = $this->configurationFactory->get();
 
+        if (!$event->isMainRequest()) {
```

Comment:
Our plugin still support sylius 1.10, that is support symfony 4.4 minimum. 

This could lead to an error on some installation. What should we do ? Update minimum version support, or add a workaround if method_exist ?

Reply:
Nice catch !
I use method_exist, great for you ?